dashboard/db/migrate/20160404233038_create_level_concept_difficulties.rb

```diff
@@ -0,0 +1,19 @@
+class CreateLevelConceptDifficulties < ActiveRecord::Migration
+  def change
+    create_table :level_concept_difficulties do |t|
+      t.references :level, index: true, foreign_key: true
+      t.timestamps null: false
+
+      t.integer :sequencing, default: 0, null: false
```

Comment:
My thinking was indeed that a level of difficulty zero doesn't encompass that concept, but I think you're right that null is a better representation.